tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -18,11 +18,13 @@
     from requests import Session
 
 
+CONSUMER_NAME: Final[str] = "great_expectations"
+PROVIDER_NAME: Final[str] = "mercury"
+
+
 PACT_MOCK_HOST: Final[str] = "localhost"
 PACT_MOCK_PORT: Final[int] = 9292
-PACT_DIR: Final[str] = str(
-    pathlib.Path(pathlib.Path(__file__).parent, "pacts").resolve()
-)
+PACT_DIR: Final[pathlib.Path] = pathlib.Path(pathlib.Path(__file__).parent, "pacts")
```

Comment:
`non-blocking`

Alternative way to build this path.
```suggestion
PACT_DIR: Final[pathlib.Path] = pathlib.Path(__file__, "..", "pacts").resolve()
```